### PR TITLE
Fix timeout for probing a game or a window capture to use

### PR DESCRIPTION
**Pull request by summeroff**

### CODE REVIEW

File: plugins/win-capture/screen-capture.c [hunk 1]
```diff
@@ -41,12 +41,13 @@ struct screen_capture {
 	int game_mode;
 	int monitor_id;
 
-	float game_window_checking;
-	float game_window_checking_interval;
+	bool  game_capture_try;
```
Comment (EddyGharbi) [inline plugins/win-capture/screen-capture.c]: I would renamed to `is_capturing_game`

File: plugins/win-capture/screen-capture.c [hunk 2]
```diff
@@ -41,12 +41,13 @@ struct screen_capture {
...
+	float game_capture_try_time;
```
Comment (EddyGharbi) [inline plugins/win-capture/screen-capture.c]: I would rename to `last_game_capture_success"

File: plugins/win-capture/screen-capture.c [hunk 3]
```diff
@@ -41,12 +41,13 @@ struct screen_capture {
...
+	float game_capture_try_interval;
```
Comment (EddyGharbi) [inline plugins/win-capture/screen-capture.c]: I would rename to `game_capture_attempts_interval`

File: plugins/win-capture/screen-capture.c (hunk 1 above)
Comment (summeroff) [reply plugins/win-capture/screen-capture.c]: names updated

**Review by EddyGharbi — APPROVED** [review]

**Pull request merged**